Makefile.am

```diff
@@ -27,6 +27,7 @@ CLEANFILES += $(dbus_service_DATA)
 CLEANFILES += $(systemduserunit_DATA)
 EXTRA_DIST += $(service_in_files)
 
+CFLAGS = -lsystemd
```

Comment:
This should become a check for libsystemd in the build system (Flatpak has a suitable check in its `configure.ac`), keeping libsystemd optional. If compiled without libsystemd support, the answer should always be "I don't know what app this is".